Sources/ParseLiveQuery.xcodeproj/project.pbxproj

```diff
@@ -891,7 +891,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
```

Comment:
I don't have a problem with it, but what's the inspiration to bump to 9.0 here?

I think we need to do it when we update to Xcode 12 anyway.

Reply:
Right, any dependency that has updated to Xcode 12 requires iOS 9 at a minimum. In this case, Starscream requires the iOS9 bump.

Reply:
Aha! I see their project file matches. 👍🏼

I only checked their podspec; it's still at 8.0.

Reply:
Actually it looks like they maybe missed updating the iOS version in the podspec. tvOS and iOS have always followed each other, haven't they?